src/core/dataframe.js

```diff
@@ -88,9 +89,52 @@ export default class Dataframe {
 
     }
 
-    _getPolygonAtPosition(pos) {
-        console.log(pos);
-        return [];
+    _getPolygonAtPosition(p) {
```

Comment:
a brief algorithm explanation will be great! (or a link)

Also maybe we could add performance tests in the future (https://github.com/speedracer/speedracer)

Reply:
Yes, I'll explain it, and yes, performance tests in the future should be added, and this is actually very easy to benchmark

Reply:
Fixed